packages/xen-api/index.mjs

```diff
@@ -6,11 +6,11 @@ import httpRequest from 'http-request-plus'
 import map from 'lodash/map.js'
 import noop from 'lodash/noop.js'
 import ProxyAgent from 'proxy-agent'
+import { cancelable, defer, fromCallback, ignoreErrors, pDelay, pRetry, pTimeout } from 'promise-toolbox'
```

Comment:
Avoid introducing unrelated changes.

Reply:
Got it. It's just an alphabetic sort of imports, it should have been in the next commit that modifies this file.